lib/auth.dart

```diff
@@ -52,10 +52,10 @@ class AccessCredentials {
   /// A refresh token, which can be used to refresh the access credentials.
   ///
   /// This field may be null.
```

Comment:
[nit] This part of the comment may be removed now that it is expressed in the signature.

Reply:
Thanks, done.